### DISCUSSION TITLE
Issues on a new Oqtane install (5.2.3)

### DISCUSSION THREAD

**Discussion by rodrigoratan** [oqtane/oqtane.framework#4720] [Q&A]
I installed Oqtane and all went well but i'm finding some issues:

After I'm logged in as a host administrator: 

If I go to Admin Dashboard > Site Settings > Database:
![image](https://github.com/user-attachments/assets/39a716bd-dcc0-426d-b4ae-40709d3fc503)
It says I'm using LocalDB but I'm using SQL server

Other issues:

![image](https://github.com/user-attachments/assets/682d7efe-f199-4ef1-97de-c2923c13692c)
Browser console log shows fail to fetch when clicking on the delete module or module reordering buttons

Module reordering: I tried to move up or move down the modules on the homepage and nothing happens after the click

Delete module from the homepage: I tried to remove the modules on the homepage (like the MIT license text) but nothing happens after the click on Delete Module

Adding a new module to a page seems to be OK.

Search: If I use the search in the header using a word that is one of the default site template pages, but it wont find the word/page

I'm using Windows Server 2019

My .NET SDKs on this machine:
C:\Users\Administrator>dotnet --list-sdks
5.0.412 [C:\Program Files\dotnet\sdk]
8.0.303 [C:\Program Files\dotnet\sdk]

And the Hosting bundle dll is at C:\Program Files\IIS\Asp.Net Core Module\V2\18.0.24262

**Comment by sbwalker** [comment]
@rodrigoratan the exact same database provider is used for LocalDB and SQL Server - which is why the UI says LocalDB. This causes no difference in run-time behavior, but I agree it is confusing.

I cannot reproduce the issue with module reordering/deletion. Is it possible that you do not have IIS configured correctly? Specifically, the WebSocket and WebDAV comfiguration is critical for Blazor (https://www.oqtane.org/blog/!/24/installing-oqtane-on-iis)

The Search Indexer is a scheduled job. On a new installation of Oqtane the scheduled jobs are configured but will not execute until after a restart of the application. So if you have not done a restart after installing the app, then no content has been indexed yet. You can check this for yourself by navigating to the Scheduled Jobs in the Admin Dashboard:

![image](https://github.com/user-attachments/assets/4a7db426-793e-4036-b65d-25595dc2bf9c)
